scalafix/input/src/main/scala/fix/Collectionstrawman_v0_LinearSeq.scala

```diff
@@ -0,0 +1,11 @@
+/*
+rule = "scala:fix.Collectionstrawman_v0"
+ */
+package fix
+
+import scala.collection.{LinearSeq, immutable}
+object Collectionstrawman_v0_LinearSeq {
+  val xs: LinearSeq[Int] = ???
+  val ys: immutable.LinearSeq[Int] = ???
+  //val zs: mutable.LinearSeq[Int] = ???
```

Comment:
a replacement for `scala.collection.mutable.LinearSeq` does not exist in strawman. Is it intentional?

Reply:
We currently have no implementation of a `mutable.LinearSeq` in the strawman.